openapi.yaml

```diff
@@ -17305,6 +17459,68 @@ components:
           x-linode-cli-color:
             None: black
             default_: white
+    LinodeConfigInterface:
+      type: object
+      description: >
+        The Network Interface to apply to this Linode's configuration profile.
+      properties:
+        label:
+          type: string
+          minLength: 0
+          maxLength: 64
+          pattern: '/[a-z0-9-]+/'
+          description: |
+            The name of this interface.
+
+            Only used for type `vlan` interfaces. Enter an empty string for type `public` interfaces.
+
+            Interface labels must be unique within the data center region.
+          example: example-interface
+        ipam_address:
+          type: string
+          format: ip/netmask
+          description: |
+            This Network Interface's private IP address in Classless Inter-Domain Routing (CIDR) notation.
+
+            Only used for type `vlan` interfaces. Enter an empty string for type `public` interfaces.
+
+            Configuring an IPAM address within the 10.0.0.0/8 block allows for automatic configuration with Network Helper. Addresses outside of this range must be configured manually.
+          example: '10.0.0.1/24'
+        purpose:
+          type: string
+          enum:
+            - public
+            - vlan
+          description: |
+            The type of interface.
+          example: vlan
+    LinodeConfigInterfaces:
+      type: array
+      items:
+        $ref: '#/components/schemas/LinodeConfigInterface'
+      description: |
+        An array of Network Interfaces to add to this Linode's configuration profile.
+
+        **Changes to Linode interface configurations can be enabled by rebooting the Linode with Network Helper enabled.**
+
+        Up to three interfaces can be entered. The position in the array determines the interface to which the settings apply:
+
+        - First/0:  eth0
+        - Second/1: eth1
+        - Third/2:  eth2
+
+        A public interface in the first/eth0 position is automatically assigned the Linode's default public IPv4 address. Only one public interface per Linode can be defined. If no public interface is defined, the Linode will not be reachable via the public internet. Access can only be established via LISH or other Linodes connected by VLAN.
```

Comment:
A public interface can be in any position, but I'm not sure about the automatic parts if it's not in position 1.